etc/templates/haproxy.conf.in

```diff
@@ -1,8 +1,8 @@
 global
   log 127.0.0.1 local6
   maxconn  ${haproxy-conf:maxconn}
-  user     ${haproxy-conf:user}
-  group    ${haproxy-conf:group}
+  # user     ${haproxy-conf:user}
+  # group    ${haproxy-conf:group}
```

Comment:
Percebam que essa mudança foi um paliativo que sugeri na lista de e-mails. As modificações que propus no #23 não precisam desta alteração no haproxy.conf.in. Visto que as variáveis definidas no settings foram incorporadas ao buildout.cfg e as alterações na documentação já corrigem o problema.

Do jeito que está atualmente, seguindo a documentação à risca, tem-se o seguinte problema no log do haproxy:

> tail -f var/log/haproxy-stderr---supervisor-whDLWJ.log

```
[ALERT] 237/203732 (2314) : [/opt/portal.buildout/parts/haproxy-build/sbin/haproxy.main()] Cannot set gid 1001
```

Conforme acima, o haproxy não consegue atribuir o grupo correspondente ao usuário corrente (plone) e não inicia. Na documentação proposta ajustei o comando de criação do usuário que evita este problema.

Daí aquele problema que o pessoal estava relatando:

> ./bin/supervisorctl status

```
haproxy                          FATAL      Exited too quickly (process log may have details)
instance1                        RUNNING    pid 2327, uptime 0:01:52
instance2                        RUNNING    pid 2337, uptime 0:01:06
zeo                              RUNNING    pid 2301, uptime 0:02:17
```